### PR TITLE
[Edit] README: Improve "How do I Contribute?" section

### PR DESCRIPTION
**Pull request by ishg-153**
### Description

Rewrote the "How do I Contribute?" section to include issue assignment step.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,12 @@ We are currently building a team of community volunteers and educators to mainta
...
 
-- Submit a Pull Request to edit an existing entry.
-- Submit a Pull Request to create a new entry.
-- Submit a typo/bug in [GitHub Issues](https://github.com/Codecademy/docs/issues).
-- Reply to the thread on [Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868) to share ideas and collaborate with other contributors, maintainers, or super users to determine good future issues.
-- Share your thoughts in this [Docs feedback form](https://codecademyready.typeform.com/to/hzVIWDgz) to tell us how we can improve Docs!
+1. Get assigned to an Issue of your interest.
+   - Ask to be assigned to one of the [available Issues](https://github.com/Codecademy/docs/issues?q=is%3Aissue+is%3Aopen+-label%3Aclaimed).
+   - Submit a typo/bug in [GitHub Issues](https://github.com/Codecademy/docs/issues).
+2. Submit a Pull Request for your assigned Issue.
```
Comment (HishamT) [inline README.md:43]: Can we include a step explicitly mentioning that you should write an entry?

I.e.: 2. write your entry

File: README.md [hunk 2]
```diff
@@ -37,11 +37,12 @@ We are currently building a team of community volunteers and educators to mainta
 
 There are many ways to contribute to Docs:
```
Comment (HishamT) [inline README.md:38]:
```suggestion
Here's how you can contribute to Codecademy Docs:
```

**Review by HishamT — CHANGES_REQUESTED** [review]
Two small changes requested, then we should be good! 😃

File: README.md (hunk 1 above)
Comment (HishamT) [reply README.md:43]: Use proper grammar of course.

**Comment by ishg-153** [comment]
@HishamT made the changes. 

Do you think we need to mention that the PR must be created on a separate branch of their fork?

**Review by HishamT — APPROVED** [review]
LGTM! 👍

**Comment by HishamT** [comment]
I don't think we need to mention that. It would make the document too long and we have a separate guide detailing that you need to create an entry on a separate fork!

**Pull request merged**